index.php

```diff
@@ -193,7 +193,7 @@
                             <button id="apply_filter" type="button" title="Применить параметры фильтра">
                                 <i class="fa fa-check" aria-hidden="true"></i>
                             </button>
-                            <button type="button" id="filter_reset" title="Сбросить параметры фильтра на значения по умолчанию">
+                            <button type="button" id="filter_reset" title="Click - cбросить параметры фильтра на значения по умолчанию. Ctrl+Click - восстановить последние сброшенные настройки.">
```

Comment:
How would you think if we add here a line separation? As for me, it looks better visually.

As [SO](https://stackoverflow.com/a/5193544) suggests, it can be done by `&#10;`:

```suggestion
                            <button type="button" id="filter_reset" title="Click - cбросить параметры фильтра на значения по умолчанию. &#10;Ctrl+Click - восстановить последние сброшенные настройки.">
```

Reply:
I think it is look better visually.

Reply:
done